html/elements/input/date.json

```diff
@@ -0,0 +1,57 @@
+{
+  "html": {
+    "elements": {
+      "input": {
+        "input-date": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/html/Element/input/date",
+            "support": {
+              "chrome": {
+                "version_added": "20"
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": "12"
+              },
+              "edge_mobile": {
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": "57"
+              },
+              "firefox_android": {
+                "version_added": "57"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": "11"
+              },
+              "opera_android": {
+                "version_added": "11"
+              },
+              "safari": {
+                "version_added": false,
+                "notes": "It is recognized but there is no UI."
```

Comment:
I realize it's from the source table, but I think this note is pretty confusing (to me it implies that it doesn't even show a text box—though I checked it and it does). Can we spell this out a little more, maybe something like "The input type is recognized, but there is no date-specific control."